features/0114-predefined-identities/README.md

```diff
@@ -92,7 +92,7 @@ In demos or tests, its genesis DID Doc looks like this:
         "serviceEndpoint": "https://localhost:23456"
     }],
     "publicKey": [{
-        "id": "key-1",
+        "id": "#key-1",
```

Comment:
This is more correct, but I still think we should use explicit URLs. For example, if this document were to be lifted and placed on a website and served from this URL: https://example.com/foo, this fragment might change from did:example:1234#key-1 to https://example.com/foo#key-1 -- and that would change the meaning of the document quite dramatically in ways that we should explore.